### PR TITLE
Bug 882140 - [calendar] progress spinner: Text is too long

### PR DESCRIPTION
**Pull request by orvi2014**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by try-server-hook** [comment]
[orvi2014 Robat Das (orvi2014)](https://github.com/orvi2014) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4347b08bcb76)

**Comment by try-server-hook** [comment]
[orvi2014 Robat Das (orvi2014)](https://github.com/orvi2014) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0cb9760cc491)

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: apps/calendar/locales/calendar.en-US.properties [hunk 1]
```diff
@@ -44,7 +44,7 @@ advanced-settings-add-account=Add Account
 field-user.placeholder=Username
 field-password.placeholder=Password
 field-full-url.placeholder=URL
-modify-account-setting-up=Please wait while your account is set up
+modify-account-setting-up=Setting up account
```
Comment (KevinGrandon) [inline apps/calendar/locales/calendar.en-US.properties]: Unfortunately for our translators to be aware of this change the key needs to change as well. You could either change it to something like ```modify-account-setting-up-2``` or ```setting-up-account```, I think either could work. (You'll also need to update the key anywhere we use it in calendar code).

Comment (orvi2014) [inline apps/calendar/locales/calendar.en-US.properties]: In `modify-account-view.css` section it's used almost every line . Do I need to update all the keys  for that ?

Comment (orvi2014) [inline apps/calendar/locales/calendar.en-US.properties]: But it looks like it's only used in  only this section . How can I find it whether it's used it or not in calendar code?

**Comment by try-server-hook** [comment]
[orvi2014 Robat Das (orvi2014)](https://github.com/orvi2014) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8f2e676ab7ad)

File: apps/calendar/locales/calendar.en-US.properties (hunk 1 above)
Comment (KevinGrandon) [inline apps/calendar/locales/calendar.en-US.properties]: I think you would need to modify it here as well: https://github.com/mozilla-b2g/gaia/blob/master/apps/calendar/elements/modify_account.html#L11

Comment (orvi2014) [inline apps/calendar/locales/calendar.en-US.properties]: Here `setting-up-account` look different. Do I need modify it too ?

**Comment by try-server-hook** [comment]
[orvi2014 Robat Das (orvi2014)](https://github.com/orvi2014) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a8be33faaf96)

**Comment by try-server-hook** [comment]
[orvi2014 Robat Das (orvi2014)](https://github.com/orvi2014) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=fec22da50404)

**Comment by try-server-hook** [comment]
[orvi2014 Robat Das (orvi2014)](https://github.com/orvi2014) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=95700dff53c6)

**Comment by try-server-hook** [comment]
[orvi2014 Robat Das (orvi2014)](https://github.com/orvi2014) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4f9c2a2ca9b7)

**Comment by try-server-hook** [comment]
[orvi2014 Robat Das (orvi2014)](https://github.com/orvi2014) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=19d670200ac6)

File: apps/calendar/locales/calendar.en-US.properties (hunk 1 above)
Comment (orvi2014) [inline apps/calendar/locales/calendar.en-US.properties]: Look at the commit file now. Is everything seems okay ?

File: apps/calendar/locales/calendar.en-US.properties [hunk 2]
```diff
@@ -44,7 +44,11 @@ advanced-settings-add-account=Add Account
 field-user.placeholder=Username
 field-password.placeholder=Password
 field-full-url.placeholder=URL
-modify-account-setting-up=Please wait while your account is set up
+<<<<<<< HEAD
```
Comment (KevinGrandon) [inline apps/calendar/locales/calendar.en-US.properties]: The new string looks fine, but you need to remove the diff markers before we can land it. Thanks.

**Pull request closed without merging**